src/components/jupyterlab-integration/instructions.js

```diff
@@ -10,18 +11,32 @@ import InstructionsDialogComponent from '../instructions-dialog';
 
 class InstructionsComponent extends Component {
   render = () => {
-    const {show, handleClose, apiKey} = this.props;
-    
-    const { protocol, hostname, port, origin } = window.location;
-
-    const command = `export GIRDER_API_KEY=${apiKey} && \\
-export GIRDER_SCHEME=${protocol.split(':')[0]} && \\
-export GIRDER_HOST=${hostname} && \\
-export GIRDER_PORT=${port} && \\
-export APP_BASE_URL=${origin} && \\
-pip install openchemistry && \\
-jupyter labextension install @openchemistry/jupyterlab && \\
-jupyter lab`;
+    const {show, handleClose, apiKey, config} = this.props;
+    const { protocol, hostname, origin } = window.location;
+
+    const commands = [
+      `export GIRDER_API_KEY=${apiKey}`,
+      `export GIRDER_SCHEME=${protocol.split(':')[0]}`,
+      `export GIRDER_HOST=${hostname}`
+    ]
+    const port = '';
```

Comment:
`port` should be coming from `window.location` instead of being initialized here, right?